cloudinit/config/cc_growpart.py

```diff
@@ -161,9 +161,13 @@ def resize(self, diskdev, partnum, partdev):
 
 class ResizeGpart(object):
     def available(self):
-        if not util.which('gpart'):
-            return False
-        return True
+        try:
+            util.subp(["gpart", "status"])
```

Comment:
how about
```python
util.which('gpart') and util.is_FreeBSD()
```
?

Reply:
The BSD `gpart` could be installed on non-FreeBSD systems or the non-BSD `gpart` could be installed on a FreeBSD system. This check handles both of those cases.

Reply:
This is what the other handler does, for reference:

https://github.com/canonical/cloud-init/blob/8116493950e7c47af0ce66fc1bb5d799ce5e477a/cloudinit/config/cc_growpart.py#L129-L140